From 770279b3792e04a979c1a333992267dd239ba4d3 Mon Sep 17 00:00:00 2001
From: Richard W.M. Jones <rjones@redhat.com>
Date: Tue, 27 May 2014 11:48:44 +0100
Subject: [PATCH 45/67] appliance: Increase udev timeout to 6000s (RHBZ#1096579).

If the image is located on NFS, and the NFS mount is performing a
large copy, then all other NFS operations stop.  This delay can be
large enough that we hit the current 10 minute timeout in udev.

Increasing this to 100 minutes is reported to cure the problem (in
reality we want udev to wait indefinitely).

(cherry picked from commit ef49b297cc56668da01efa637fb80007a76d23dd)
---
 src/launch.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/launch.c b/src/launch.c
index 1583075..caf1c6f 100644
--- a/src/launch.c
+++ b/src/launch.c
@@ -340,7 +340,7 @@ guestfs___appliance_command_line (guestfs_h *g, const char *appliance_dev,
      " noapic"                  /* workaround for RHBZ#857026 */
 #endif
      " console=" SERIAL_CONSOLE /* serial console */
-     " udevtimeout=600" /* good for very slow systems (RHBZ#480319) */
+     " udevtimeout=6000"/* for slow systems (RHBZ#480319, RHBZ#1096579) */
      " no_timer_check"  /* fix for RHBZ#502058 */
      "%s"               /* lpj */
      " acpi=off"        /* we don't need ACPI, turn it off */
-- 
1.7.1

